testdata/cert.pem

```diff
@@ -0,0 +1,21 @@
+-----BEGIN CERTIFICATE-----
+MIIDZTCCAk2gAwIBAgIUREQdo6KovJtZqBw/fZWw4imnDPkwDQYJKoZIhvcNAQEL
+BQAwQjELMAkGA1UEBhMCWFgxFTATBgNVBAcMDERlZmF1bHQgQ2l0eTEcMBoGA1UE
+CgwTRGVmYXVsdCBDb21wYW55IEx0ZDAeFw0yMTA5MjIxMDU4MjRaFw0zMTA5MjAx
+MDU4MjRaMEIxCzAJBgNVBAYTAlhYMRUwEwYDVQQHDAxEZWZhdWx0IENpdHkxHDAa
+BgNVBAoME0RlZmF1bHQgQ29tcGFueSBMdGQwggEiMA0GCSqGSIb3DQEBAQUAA4IB
+DwAwggEKAoIBAQC9N9BW1tV0pRjchNIvPhNsMLpp0OKACjpWhPHL/sk7aB8v5mre
+9m5LiKFIiWJmEVC8X5ZNXfvGVPWEn7Fh5ywp6/IKVxeG0KWdqGXa8WcZJTvHvjUq
+Sm7R9z85Pl1ZrrDAWYCW9UHqbwdL2EloAyfxGDRzb2mCXUIoshTQHd94fB1gL9pj
+dWFJe/yxvLiisZM2Et/oMFPU6rE+IX/mA+b8VzknLeiMYYzBmmkn9EWhhVura/VK
+nPOae1m+nj6bxj+tgl83MU96QMaD8MCJawL6YzixFX56Fh6hfwHOy1Gwoyc0NxXM
+kxyXJDrvEgDPKTNdcT1wm7+i562NL1kupD73AgMBAAGjUzBRMB0GA1UdDgQWBBSp
+fG4Foem3RyvuQDkorxH6ZhRZ3zAfBgNVHSMEGDAWgBSpfG4Foem3RyvuQDkorxH6
+ZhRZ3zAPBgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQCFNE6GW4+y
+wxXAXiNNcw4/F9ytTmwz9pBLN2KEpjfqk2d7iyDWJKwtChVjadU3L5dNV9UKOJM7
+n5OG/rJSh82gjnRlgWIiR09FPZQoBH3WiGehnSdOcJmkQUcnIPuXN2TkWVUqFZcy
+1y1vcule0mLKjGH+ejwqbv7LgkS/i/AlGpeqavEXFi5IEAlXZbyJ9MT2j1eygMmj
+XQ8WgQb1DzDlSGiOawdLAP4ULTIaVgx4SnH/4+g51CPH8Aj69eYPcsRMKdjUxAwn
+LbFzsderXnoHddvHoWSEsOX43kikoSlGjW3p/vpWXhNQJiH9eYCSSHmIDcl8CCNt
+6DLxiUiQf33S
+-----END CERTIFICATE-----
```

Comment:
Probably @tisnik knows better, but probably this file should be included in a `.gitleaks.toml` file in the root directory of the repository. You can see an example in Smart Proxy.
That file is intended to provide an "allow list" for certificates/secrets exposed in repositories to a scanner of this kind of data leaks run by Red Hat.